examples/coupling_edwards_teng/000_coupling.py

```diff
@@ -0,0 +1,99 @@
+import xtrack as xt
+import numpy as np
+import matplotlib.pyplot as plt
+from cpymad.madx import Madx
+import xobjects as xo
+
+mad = Madx()
+mad.call('../../test_data/lhc_2024/lhc.seq')
+mad.call('../../test_data/lhc_2024/injection_optics.madx')
+
+mad.beam()
+mad.use('lhcb1')
+
+mad.globals.on_x1 = 0
+mad.globals.on_x2h = 0
+mad.globals.on_x2v = 0
+mad.globals.on_x5 = 0
+mad.globals.on_x8h = 0
+mad.globals.on_x8v = 0
+
+mad.globals.on_sep1 = 0
+mad.globals.on_sep2h = 0
+mad.globals.on_sep2v = 0
+mad.globals.on_sep5 = 0
+mad.globals.on_sep8h = 0
+mad.globals.on_sep8v = 0
+
+mad.globals.on_a2 = 0
+mad.globals.on_a8 = 0
+
+mad.globals['kqs.a67b1'] = 1e-4
+
+twmad = mad.twiss()
+
+line = xt.Line.from_madx_sequence(mad.sequence.lhcb1, deferred_expressions=True)
+line.particle_ref = xt.Particles(p0c=450e9)
+tw = line.twiss4d(compute_coupling_elements_ed_teng=True)
+
+ngtw = line.madng_twiss()
+
+r11_mad, r12_mad, r21_mad, r22_mad = twmad.r11, twmad.r12, twmad.r21, twmad.r22
+s_mad = twmad.s
+r11_ng, r12_ng, r21_ng, r22_ng = ngtw.r11_ng, ngtw.r12_ng, ngtw.r21_ng, ngtw.r22_ng
+s_ng = ngtw.s
+
+r11_mad_at_s = np.interp(tw.s, s_mad, r11_mad)
+r12_mad_at_s = np.interp(tw.s, s_mad, r12_mad)
+r21_mad_at_s = np.interp(tw.s, s_mad, r21_mad)
+r22_mad_at_s = np.interp(tw.s, s_mad, r22_mad)
+
+sgn_ng = np.sign(tw.r11 / r11_ng)
+
+# Compare results with MAD-X
+plt.close('all')
+fig, ((ax0, ax1), (ax2, ax3), (ax4, ax5)) = plt.subplots(3, 2, figsize=(10, 12), sharex=True)
+ax0.plot(s_mad, r11_mad, label='MAD-X')
+ax0.plot(s_ng, (r11_ng * sgn_ng), label='MAD-NG', linestyle='--')
+ax0.plot(tw.s, (tw.r11.real), label='Xsuite', linestyle=':')
```

Comment:
`r11` can get confused with the R matrix we already have. I propose to use `r11_edw_teng`